models/sudoswap/ethereum/sudoswap_ethereum_pool_settings_latest.sql

```diff
@@ -0,0 +1,168 @@
+{{ config(
+        alias = 'pool_settings_latest',
+        partition_by = ['pool_address'],
```

Comment:
similar idea here

Reply:
although the test is failing on this model for unique, so it's not exactly the same as the other

Reply:
Ok, this should be unique.
Any idea why the general unique tests don't show up in the dbt slim ci?

Reply:
ah, ~~test~~ -> tests. got it